src/OpenTelemetry.Instrumentation.Owin/Implementation/DiagnosticsMiddleware.cs

```diff
@@ -165,7 +165,7 @@ private static void RequestEnd(IOwinContext owinContext, Exception exception, lo
                 {
                     activity.SetStatus(Status.Error);
 
-                    if (OwinInstrumentationActivitySource.Options.RecordException)
+                    if (OwinInstrumentationActivitySource.Options!.RecordException)
```

Comment:
`OwinInstrumentationActivitySource.Options` is marked nullable [here](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/1601/files#diff-f6849c8742fbbf792696bfc7879614003688375980d604ffcca6145c886bb28dR20).
```suggestion
                    if (OwinInstrumentationActivitySource.Options?.RecordException)
```

Reply:
e445962e2ca5b5cec27c4f3358c9603ab43c3242

Options should be never null, if instrumentation is configured by standard extension method. I do not see anything but with defensive programming. Changed both comments.

Reply:
Good catch. Already done.

Reply:
Could you update that here as well? https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/main/src/OpenTelemetry.Instrumentation.Owin/Implementation/DiagnosticsMiddleware.cs#L126-L130

Reply:
Fixed 8740c4fe6bafca0cc10efda2ebc1e084a9d26a11